config/src/main/java/org/axonframework/config/EventProcessingModule.java

```diff
@@ -129,6 +130,8 @@
     protected final Map<String, PooledStreamingProcessorConfiguration> psepConfigs = new HashMap<>();
     protected final Map<String, DeadLetteringInvokerConfiguration> deadLetteringInvokerConfigs = new HashMap<>();
 
+    protected final List<BiFunction<Configuration, String, SequencedDeadLetterQueue<EventMessage<?>>>> deadLetterProviders = new ArrayList<>();
```

Comment:
Why allow the registration of several factories? Shouldn't this represent the default factory and that's it?

And, a nit: I would similarly rename this field to `deadLetterQueueProviders`.

Reply:
A single one would keep it easier. Also it's clear that it can be overriden by calling a method after the auto config.

Reply:
It's single now.